files/scripts/swss.sh

```diff
@@ -321,6 +315,22 @@ function check_macsec()
     fi
 }
 
+function check_ports_present()
+{
+    PORT_CONFIG_INI=/usr/share/sonic/device/$PLATFORM/$HWSKU/$DEV/port_config.ini
+    HWSKU_JSON=/usr/share/sonic/device/$PLATFORM/$HWSKU/$DEV/hwsku.json
+
+    if [[ -f $PORT_CONFIG_INI ]] || [[ -f $HWSKU_JSON ]]; then
```

Comment:
Should we check for presence of "interfaces" in platform.json file? Does all platforms need to define hwsku.json file

Reply:
@judyjoseph no all platforms need not define hwsku.json file. Check is if either one of the file (port_config.ini or platform.json ) present then we have have front panel ports on the system.  Current behaviour assume this files are define only when there are Ethernet ports on the system